aptos-move/framework/aptos-token-objects/sources/aptos_token.move

```diff
@@ -68,6 +69,14 @@ module aptos_token_objects::aptos_token {
         property_mutator_ref: property_map::MutatorRef,
     }
 
+    struct TokenUpdatePermission has copy, drop, store {
+        token_address: address,
+    }
```

Comment:
whether we want it here or not, from expressiviness of permissions model, we should think of how would it be implemented to have either a global permission to update all tokens, or a specific permission to update a concrete object. 

in that case would these be enums, we would be checking both against? or something else more concrete with permissioned_signer?

Reply:
second I don't think this is the file you want to be adding permissions to - this is an example implementation of tokens on aptos. 
aptos-move/framework/aptos-token-objects/sources/token.move and collection.move are the right places